notifications.py

```diff
@@ -0,0 +1,16 @@
+def median_list(nums):
+    nums = sorted(nums)
+    if len(nums) % 2 == 1:
+        return nums[len(nums)//2]
+    else:
+        return int((nums[len(nums)//2 + 1] + nums[len(nums)//2 - 1])/2.0)

```

Comment:
This construct: int(<some_integer> / 2.0) should make you stop. You're doing 2 casts on this line:
- casting <some_integer> to float for the division with 2.0.
- casting the resulting float to int.

This should be written: <some_integer> / 2 (integer division, which is likely what you want unless you really want the int() rounding).

Your computation of the median on this line has 2 bugs:
- The indexes are wrong.
- The median doesn't have to be an int (which is slightly confusing)